float64[] position
bool EndEffector_Status